Cargo.toml

```diff
@@ -143,6 +143,7 @@ tokio-cron-scheduler = { version = "0.13.0", default-features = false }
 tokio-stream = { version = "^0.1", default-features = false }
 pin-project-lite = "0.2.7"
 tower = { version = "0.5.2", default-features = false }
+async-trait = { version = "0.1.88", default-features = false }
```

Comment:
We should only be using native async traits and `dynosaur` if you need trait objects